packages/force_update/README.md

```diff
@@ -3,12 +3,9 @@
 This is an internal Leancode package for quickly implementing Force Update. To use it, just wrap your `MaterialApp` in a `ForceUpdateGuard`
 widget and pass an instance of CQRS.
 
-# Behavior
+# Behaviour
 
-On every app launch, a request will be made to fetch the minimum required version of the app. To avoid
-affecting UX, the app will not wait for the request to complete. Instead, the result will be stored in prefs and used on next app launch. 
-If the app notices on startup that the force updated is required, a force update screen will be shown. If the update is suggested (but not required), a dialog will be shown.
-To keep the minimum required version up to date, requests are made every 5 minutes when the app is running.
+On every app launch, a request will be made to fetch the minimum required version of the app. By default, the result will be applied immediately after the response is obtained - if necessary, the package will display either force update screen or a dialog suggesting to update the app. If you do not want to interrupt the user after they have launched the app (for instance, they could have launched it without internet connection and then turn it on at some point) and would rather only show a force update screen or a dialog on app launch, you can change the behaviour of the package to make it store the responses and apply them on subsequent app launches. This can be done by setting `showForceUpdateScreenImmediately` or `showSuggestUpdateDialogImmediately` to false in `ForceUpdateGuard`. Regardless of those settings, to keep the minimum required version up to date, requests are made every 5 minutes while the app is running.
```

Comment:
Use en dashes instead of hyphens when appropriate

Reply:
does "when appropriate" mean every time in this case?

Reply:
Yes, it would seem so 😄